config/index.json.txt

```diff
@@ -2,6 +2,8 @@
   "db": "postgres://user:password@server/database",
   "cache": 3600,
   "port": 3000,
+  "cors_origin_whitelist": [],

```

Comment:
can we change this to `fastifyCorsOptions` and set its value to `{}`? This would give us the flexibility of adding our own object.